sympy/functions/elementary/complexes.py

```diff
@@ -422,6 +474,12 @@ class Abs(Function):
     x**2
     >>> abs(-x) # The Python built-in
     Abs(x)
+    >>> Abs(3*x + 2*I)
+    sqrt(9*x**2 + 4)
+    >>> Abs(8*I)
+    8
+    >>> Abs(-x).fdiff()
```

Comment:
fdiff is not meant to be used by end users.

Reply:
I see. Removed.